apps/sms/index.html

```diff
@@ -145,7 +145,7 @@ <h1 id="threads-edit-mode" data-l10n-id="selectThreads-title"></h1>
           <div class="bottom-tab" role="region" >
             <ul class="bb-tablist">
               <li role="presentation">
-                <button id="threads-delete-button" data-action="delete-button"></button>
+                <button id="threads-delete-button" class="disable-when-dialog-animates" data-action="delete-button"></button>
```

Comment:
I think you're missing the point here.

If we use a "disabled" attribute, we'd have to hunt down all places where we need to add/remove it, from the JS script that does the animation.

Here we add a class in the markup, and we never ever touch it. We simply add/remove a class on body which will act on elements on this class.

This is a lot more elegant, more maintainable, and the behavior can be shared with the various dialog libs we have.